src/main/java/com/apuope/apuope_re/services/RegistrationService.java

```diff
@@ -15,6 +16,8 @@ public class RegistrationService {
     private final DSLContext dslContext;
     private final UserRepository userRepository;
 
+    private final BCryptPasswordEncoder passwordEncoder = new BCryptPasswordEncoder(10);
```

Comment:
We have this same specified in our own PasswordHashService, so it could be used here.